projects/hip/cmake/FindHIP.cmake

```diff
@@ -97,9 +97,14 @@ if(NOT APPLE)
     # Search for HIP installation
     if(NOT HIP_ROOT_DIR)
         # Search in user specified path first
+        if(WIN32)
+            set(HIPCONFIG_EXE_NAME bin/hipconfig.exe)
+        else()
+            set(HIPCONFIG_EXE_NAME bin/hipconfig)
+        endif()
```

Comment:
Prefer `${CMAKE_EXECUTABLE_SUFFIX}` (https://cmake.org/cmake/help/latest/variable/CMAKE_EXECUTABLE_SUFFIX.html) over a conditional like this.

We had a patch in TheRock that solved this already: https://github.com/ROCm/TheRock/blob/main/patches/amd-mainline/rocm-systems/0007-Fix-FindHIP.cmake-search-for-hipconfig.patch